tcms/testcases/forms.py

```diff
@@ -104,7 +104,24 @@ def populate(self, product_id=None):
             self.fields['category'].queryset = Category.objects.all()
 
 
-class NewCaseForm(BaseCaseForm):
+class NewCaseForm(BaseCaseForm, forms.ModelForm):
```

Comment:
don't use multiple inheritance. This should inherit only from ModelForm. B/c the original form is used in several places create a new class that will be used only in views.